applications/WindEngineeringApplication/python_scripts/test_suite.py

```diff
@@ -0,0 +1,27 @@
+# Kratos imports
+import KratosMultiphysics.KratosUnittest as UnitTest
+
+# STL imports
+from enum import Enum
+
+
+class SuiteFlags(Enum):
```

Comment:
uppercase?

Reply:
I changed them to uppercase, but we'll have to keep this in mind because the keys in the [global kratos test suite map](https://github.com/KratosMultiphysics/Kratos/blob/3380733c37c889ef9d28387ed6183bcab12e51c7/kratos/python_scripts/KratosUnittest.py#L193) are all lowercase.